src/app/js/lib/stylesToClassName.js

```diff
@@ -0,0 +1,14 @@
+import { StyleSheet, css } from 'aphrodite/no-important';
+
+export default (styles, componentName) => {
+    const prefix = componentName ? `${componentName}-` : '';
+    const stylesheet = StyleSheet.create(styles);
+    return Object.keys(stylesheet).reduce(
+        (acc, classname) =>
+            Object.defineProperty(acc, classname, {
+                get: () =>
+                    `${css(stylesheet[classname])} ${prefix}${classname}`,
```

Comment:
```suggestion
                    `${css(stylesheet[classname])} ${prefix}-${classname}`,
```

Reply:
prefix is either `''` or `something-`, this way it can be optional